tests/resources/helper_functions.py

```diff
@@ -213,11 +213,14 @@ def verify_supply(mint_amount=to_wei(50)):
 
 
 def deploy_erc721_erc20(
-    web3: Web3, config: Config, erc721_publisher: Wallet, erc20_minter: Wallet
+    web3: Web3,
+    config: Config,
+    erc721_publisher: Wallet,
+    erc20_minter: Wallet,
+    cap=500000000000000000,
```

Comment:
maybe we can use web3.toWei("0.05", "ether") instead of putting this large numbers.